src/main/resources/api-specs/or/objects-openapi.yaml

```diff
@@ -1044,28 +1101,26 @@ components:
           type: boolean
           description: Use field-based authorization
         fields:
-          type: object
-          additionalProperties: {}
```

Comment:
Another occasion where `type` and `additionalProperties` were removed, probably the same cause. There are more examples to be found in the Objecttypes API specs. 

Should we manually add it here too?

Reply:
Hmm, most likely these are fields which we do not use then. Our backend code compiles so we can be pretty sure that this is compatible with our current code. So I would leave these.